eclipse-collections-code-generator/src/main/resources/impl/map/mutable/mutableObjectPrimitiveMapFactoryImpl.stg

```diff
@@ -49,6 +49,54 @@ public class MutableObject<name>MapFactoryImpl implements MutableObject<name>Map
         return this.empty();
     }
 
+    @Override
+    public \<K> MutableObject<name>Map\<K> of(K key, <type> value)
+    {
+        return this.with(key, value);
+    }
+
+    @Override
+    public \<K> MutableObject<name>Map\<K> with(K key, <type> value)
+    {
+        return Object<name>HashMap.newWithKeysValues(key, value);
+    }
+
+    @Override
+    public \<K> MutableObject<name>Map\<K> of(K key1, <type> value1, K key2, <type> value2)
+    {
+        return Object<name>HashMap.newWithKeysValues(key1, value1, key2, value2);
```

Comment:
We should be consistent with implementing in `with` and then having `of` call `with`.